packages/livechat/package.json

```diff
@@ -112,5 +112,8 @@
 		"updateFiles": [
 			"package.json"
 		]
+	},
+	"volta": {
```

Comment:
Are these changes required for this to work or just QoL Improvements? If they are we could do this on a different PR :)

Reply:
😢 no...